### ISSUE TITLE
Palworld - Saves converting out of date (June)

### ISSUE THREAD

**Issue by bullfrogberry** [Z1ni/XGP-save-extractor#169]
As of June 27th Palworld has added an option to host dedicated servers for xbox gamepass. it uses the same save and file structure as the steam version now. But some time ago Palworld changed how they saved the world files. When I and others use this XGP extractor with Palworld it is an old save from months ago. I have been seeing a possible solution using https://github.com/cheahjs/palworld-save-tools but also have been unsuccessful. Can the XGP extractor be updated to find a solution to fix this problem?

**Comment by osterac** [comment]
Yeah, it's not working for me either. Converted save won't load, game hangs, empty log file.

**Comment by bullfrogberry** [comment]
I don't have as much technical knowledge as I would like on this subject but what I can figure that this XGP extractor is not completing the new updated level file in the level folder correctly/corrupting it, or the file truly needs to be converted to a JSON and back and renamed to level.sav and that program cant understand the new world format.

**Comment by Silvers12** [comment]
Same here, i don't have level.sav but Level\01.sav ... Rename the file doesn't work.

**Comment by Z1ni** [comment]
So if you take the extracted `Level/01.sav` file from the folder with the other files and rename it to `Level.sav`, it doesn't work?

**Comment by abeasst** [comment]
So you don’t even get the 01.sav in save extracts. It doesn’t extract any
files that are up to date in palworld. The latest extracts I’ve been able
to get are from April.

**Comment by Z1ni** [comment]
> So you don’t even get the 01.sav in save extracts. It doesn’t extract any files that are up to date in palworld. The latest extracts I’ve been able to get are from April.

Does it extract any files? What files are there? I'm understanding from the other comments that there's still `Level\01.sav`. Can you zip up your wgs folder under `%LOCALAPPDATA%\Packages\PocketpairInc.Palworld_ad4psfrxyesvt\SystemAppData` and share it?

**Comment by abeasst** [comment]
It doesnt extract anything past 4-22. All save files after that date don'e get extracted at all.
[wgs.zip](https://github.com/user-attachments/files/16075096/wgs.zip)

**Comment by Z1ni** [comment]
> It doesnt extract anything past 4-22. All save files after that date don'e get extracted at all. [wgs.zip](https://github.com/user-attachments/files/16075096/wgs.zip)

Do you play on a server? It seems that your recent wgs files don't contain anything but the `LocalData.sav` files. If the game doesn't download the level data etc. to your computer, there's nothing the save extractor can do.
If those saves are local, I don't know where the game is storing the level data, as it's seemingly not in the wgs data as per your files.

**Comment by abeasst** [comment]
I play on xbox. This is extracting from my computer that uses the xbox app.

**Comment by RPMC-Marco** [comment]
I am getting the same problem as well, I was playing locally with 2 friends and am now setting up a dedi server, however when I extract the game from my game pass PC and move it over, the save is from months ago

**Comment by 13adman** [comment]
I have been having this problem since the update too. The tool extracts all the info for LevelMeta.Sav , LocalData.sav and the folders for players and level are there, but the Level.sav file is missing. if I run the tool on an old world that I haven't played since the update it pulls through all the info and can be successfully ran on the steam version. It seems that something within the update has changed the way the Xbox saves work and is preventing the tool from generating all the info.

**Comment by xoanaaa** [comment]
I have the same problem too

**Comment by kinghawn** [comment]
seems like everyone's having the same problem with the "level.sav" file not appearing. hopefully they can fix this soon.

**Comment by OtroGaato** [comment]
i found a fix involving a conversion of the file 01.sav found in Levels Folder
https://www.reddit.com/r/Palworld/comments/1c04l6t/comment/kyui2be/
i just tried and works great

**Comment by osterac** [comment]
> i found a fix involving a conversion of the file 01.sav found in Levels Folder https://www.reddit.com/r/Palworld/comments/1c04l6t/comment/kyui2be/ i just tried and works great

Worked for me. Thanks!

**Comment by Asran74** [comment]
https://www.reddit.com/r/Palworld/comments/1c04l6t/comment/kyui2be/

doesn't work for me,

the entire save is previous, whether it is the pals, the saving of the world and its camps, only the appearance of the character (physical and not equipment) and the skill tree are good

Oddly I still unlock recent successes like the oil platform, while a large part of the backup predates this exploit

**Comment by CacheCream** [comment]
I am having the same issue! I really hope someone finds a solution.

**Comment by NC12001** [comment]
After 6 hours somehow I was able to transfer my xbox gamepass save to steam without it being outdated, but after a hour trying to recreate whatever solution I did in a video, I just could not do it again.  

What I can tell you guys is that I was able to get my current save from my xbox gamepass palworld doing these steps in the video below. 

The problem is putting these game files into steam, what happens is that it will force me to make a new character when I load my xbox save into steam. After hours of doing random things like replacing the Level.sav and everything in different orders after a while something just worked and my save was loading and not forcing me to make a new character. Very sorry for not knowing what I did but hopefully you guys can make use of the video I made.


TLDR: The video below will show you how to get ur current palworld save and not a outdated save. As to how to get this save to work with steam I have no idea, I made it work with my steam but I forgot what steps I took to get it to work (without it making me make a new character upon loading my world) 


Feel free to ask any questions


https://github.com/user-attachments/assets/fb02e30a-249f-4b07-b391-ff5f279a1469
